pkg/kubectl/cmd/explain.go

```diff
@@ -106,19 +105,20 @@ func RunExplain(f cmdutil.Factory, out, cmdErr io.Writer, cmd *cobra.Command, ar
 	}
 
 	if len(apiVersionString) == 0 {
-		groupMeta, err := scheme.Registry.Group(gvk.Group)
-		if err != nil {
-			return err
+		if len(gvk.Version) == 0 {
+			groupMeta, err := scheme.Registry.Group(gvk.Group)
```

Comment:
pre-existing, but this comand should not have any dependency on a compiled in scheme.  @juanvallejo see about removing this?